Documentation/bpf.rst

```diff
@@ -3354,7 +3354,376 @@ the toolchain section under the respective tools.
 tc (traffic control)
 --------------------
 
-TODO
+Aside from other program types such as XDP, BPF can also be used out of the
+kernel's tc (traffic control) layer in the networking data path. On a high-level
+there are three major differences when comparing BPF XDP programs to tc BPF
+ones:
```

Comment:
`tc BPF` should be `BPF tc` to follow the other one, perhaps?

Reply:
Should say XDP BPF